### ISSUE TITLE
meshctl install fails with common.cluster: String length must be greater than or equal to 1

### ISSUE THREAD

**Issue by elinesterov** [solo-io/gloo#9928]
### Gloo Edge Product

Enterprise

### Gloo Edge Version

2.6

### Kubernetes Version

v1.29.2

### Describe the bug

Following the quickstart guide for single cluster https://docs.solo.io/gloo-mesh-enterprise/latest/getting_started/single/gs_single/

the key is valid
```
meshctl license check --key $(echo ${GLOO_MESH_LICENSE_KEY} | base64 -b0)
 INFO  License key #0 for product gloo is valid. Expires at 18 Sep 24 15:48 PDT
 INFO  License key #0 for product gloo-mesh is valid. Expires at 18 Sep 24 15:48 PDT
 INFO  License key #0 for product gloo-core is valid. Expires at 18 Sep 24 15:48 PDT
 INFO  License key #0 for product gloo-mesh-gateway is valid. Expires at 18 Sep 24 15:48 PDT
 INFO  License key #0 for product gloo-network is valid. Expires at 18 Sep 24 15:48 PDT
 INFO  License key #0 for product gloo-gateway is valid. Expires at 18 Sep 24 15:48 PDT
 INFO  License key #0 for product gloo-trial is valid. Expires at 18 Sep 24 15:48 PDT
 SUCCESS  Licenses are valid.
 ```
 
 Running the command:
 ```
  meshctl install --profiles gloo-mesh-single,ratelimit,extauth \
--set common.cluster=$CLUSTER_NAME \
--set glooMgmtServer.createGlobalWorkspace=true \
--set licensing.glooMeshLicenseKey=$GLOO_MESH_LICENSE_KEY
 INFO  💻 Installing Gloo components in the management cluster
 SUCCESS  Finished downloading chart.
 SUCCESS  Finished upgrading chart 'gloo-platform-crds' as release gloo-mesh:gloo-platform-crds
 SUCCESS  Finished downloading chart.
 ```
 
 Got the error:
 ```
   ERROR   installing gloo-platform: installing helm chart: values don't meet the specifications of the schema(s) in the following chart(s):
          gloo-platform:
          - common.cluster: String length must be greater than or equal to 1
  ```
  

### Expected Behavior

installation is not failing

### Steps to reproduce the bug

Follow the doc ttps://docs.solo.io/gloo-mesh-enterprise/latest/getting_started/single/gs_single/

**Comment by timflannagan** [comment]
@elinesterov Awesome to see folks already tinkering with the 2.6 GME release. Just a quick heads-up: this repository is for the gateway project/product, so it might be a bit tricky to get attention on this issue here. cc @Sodman @marcogschmidt for GME awareness.

**Comment by Sodman** [comment]
Hi @elinesterov, from the error message it looks like `CLUSTER_NAME` environment variable might not have been set in this context? You can set it to any arbitrary value for what you want to name the cluster, e.g. "mgmt".

**Comment by elinesterov** [comment]
@timflannagan, sorry about that. Where should I file the issue? I didn't see other options when I opened an issue here.

@Sodman, I even tried using `--set common.cluster=gloo-mesh` explisetly

In another part of the documentation, I found that `meshctl license check --key $(echo ${GLOO_MESH_LICENSE_KEY} | base64 -w0)` doesn't work on Macos because you should run `base64 -b0` on Macos. Could there be a chance that the `meshctl install` issue is related to Macos only?

**Comment by Sodman** [comment]
Hi @elinesterov this has been fixed and released in Gloo Platform v2.6.1.

Upon investigating further, we found the meshctl `--set` flag was broken in v2.6.0 in a bad merge. Using the `--profiles` was also impacted in a similar manner. No other meshctl versions other than v2.6.0 were affected by this bug. Passing a helm values file to meshctl, or installing with helm directly, were not impacted by this bug.

Closing this issue out now, feel free to reach out if you run into any other issues!
